packages/components/CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - Wrong validation during SSR hydration of `Link Tile` and `Select Wrapper`
   ([#2588](https://github.com/porsche-design-system/porsche-design-system/pull/2588))
+- `Modal` scrollable modal does not jump to top on changes within dialog
```

Comment:
Perhaps also mention as separate bullet point that unnecessary lifecycles are prevented when prop values don't change for complex values.